### PR TITLE
Fixed TTK samples issues

### PR DESCRIPTION
**Pull request by Pawank-MSFT**

### CODE REVIEW

File: samples/meetings-live-caption/nodejs/index.js [hunk 1]
```diff
@@ -28,9 +23,19 @@ server.engine('html', require('ejs').renderFile);
 server.set('view engine', 'ejs');
 server.set('views', __dirname);
 
-const port = process.env.port || process.env.PORT || 3978;
-server.listen(port, () => 
-    console.log(`\n${server.name} listening to http://localhost:${port}`)
+// Specify the paths to your SSL certificate and private key
+const sslOptions = {
+    key: fs.readFileSync(process.env.SSL_KEY_FILE),
```
Comment (Prithvi-MSFT) [inline samples/meetings-live-caption/nodejs/index.js:28]: are we giving certificate values in env file ?

Comment (Pawank-MSFT) [reply samples/meetings-live-caption/nodejs/index.js:28]: yes

**Comment by Pawank-MSFT** [comment]
Executed build pipeline:
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/7899112543

**Review by Harikrishnan-MSFT — APPROVED** [review]
Looks Good. Approving!

**Review by Prithvi-MSFT — APPROVED** [review]
approved

**Comment by Pawank-MSFT** [comment]
Executed build pipeline:
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8019069680

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]

File: samples/app-checkin-location/nodejs/views/CheckIn.ejs [hunk 2]
```diff
@@ -29,9 +30,31 @@
...
+                }
+            // Don't do anything if the permission was denied.
+            });
+            if (navigator.geolocation) {
+                navigator.geolocation.getCurrentPosition(showPosition);
+            } else {
+                x.innerHTML = "Geolocation is not supported by this browser.";
+            }
+        }
+        function showPosition(position) {
```
[the remark this answers is not in the document]
Comment (Pawank-MSFT) [reply samples/app-checkin-location/nodejs/views/CheckIn.ejs:49]: Done

File: samples/app-checkin-location/nodejs/views/ViewLocation.ejs [hunk 3]
```diff
@@ -26,19 +26,34 @@
...
+            if (microsoftTeams.location.isSupported()) {
+                microsoftTeams.location.showLocation(location, (error, result) => {
+                    if (error) {
+                        let errorMessage = "Error";
+                        if (error.message) {
+                            errorMessage = `Error Code: ${error.errorCode}, Message: ${error.message}`;
+                        } else {
+                            errorMessage = `Error Code: ${error.errorCode}`;
+                        }
+                        alert(errorMessage);
```
[the remark this answers is not in the document]
Comment (Pawank-MSFT) [reply samples/app-checkin-location/nodejs/views/ViewLocation.ejs:38]: We should keep this

**Comment by Pawank-MSFT** [comment]
Executed build pipeline:
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8092907184

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine.

**Pull request merged**